### ISSUE TITLE
animationDuration doesn't seem to work.

### ISSUE THREAD

**Issue by JustinWestphal** [joaopereirawd/animatedModal.js#10]
I don't seem to be able to get a reaction using the animationDuration option. Could you double check that it works, and if so, maybe provide a usage example?

**Comment by joaopereirawd** [comment]
Hello Justin, I will check your question as soon possible. 

Thanks for your feedback.

**Comment by idrolitino** [comment]
Hi, on chrome and safari works, in firefox no.
I state that I am not an expert. 
However I have seen that animate.duration is currently only set for webkit.
I have solved so:
in animatedModal.js you see, in "var = initStyles", last line:
'-webkit-animation-duration': settings.animationDuration,
Or changing this line
'animation-duration': settings.animationDuration,
(and this should apply to all browsers)
Or adding later also
'-moz-animation-duration': settings.animationDuration,
and also other specific statements for other browsers.

newby question: I have tried to beautify online animatedModal.min.js, but the result does not seem equal to animatedModal.js
Can I use animatedModal.js or animatedModal.min.js has more functionality?
Thank you.
